.github/workflows/release.yml

```diff
@@ -42,6 +42,10 @@ env:
   # PRs opened from fork and from dependabot don't have access to repo secrets
   HAS_ACCESS_TO_GITHUB_TOKEN: ${{ github.event_name != 'pull_request' || (github.event.pull_request.head.repo.full_name == github.repository && github.actor != 'dependabot[bot]') }}
 
+permissions:
+  # create comments on commits for docker images needs the `write`` permission
```

Comment:
If the write permission is only needed in a specific job, probably it's better define the permissions for that particular job instead for all jobs. https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

Reply:
done